files/default/neutron-ha-tool.py

```diff
@@ -270,50 +272,39 @@ def l3_agent_migrate(qclient, noop=False, now=False):
     LOG.info("There are %s offline L3 agents and %s online L3 agents",
              len(agent_dead_list), len(agent_alive_list))
 
-    if len(agent_dead_list) > 0:
-        if len(agent_alive_list) < 1:
-            LOG.exception("There are no l3 agents alive to migrate "
-                          "routers onto")
-
-        timeout = 0
-        if not now:
-            while timeout < TAKEOVER_DELAY:
-
-                agent_list_new = list_agents(qclient)
-                agent_dead_list_new = agent_dead_id_list(agent_list_new,
-                                                         'L3 agent')
-                if len(agent_dead_list_new) < len(agent_dead_list):
-                    LOG.info("Skipping router failover since an agent came "
-                             "online while ensuring agents offline for %s "
-                             "seconds", TAKEOVER_DELAY)
-                    sys.exit(0)
-
-                LOG.info("Agent found offline for seconds=%s but waiting "
-                         "seconds=%s before migration",
-                         timeout, TAKEOVER_DELAY)
-                timeout += 1
-                time.sleep(1)
-
-        for agent_id in agent_dead_list:
-            LOG.info("Querying agent_id=%s for routers to migrate", agent_id)
-            router_id_list = list_routers_on_l3_agent(qclient, agent_id)
+    if len(agent_dead_list) == 0:
+        return
 
-            for router_id in router_id_list:
+    if len(agent_alive_list) < 1:
+        LOG.error("There are no l3 agents alive to migrate routers onto - "
+                  "aborting!")
+        sys.exit(1)
 
-                target_id = random.choice(agent_alive_list)
-                LOG.info("Migrating router=%s to agent=%s",
-                         router_id, target_id)
+    timeout = 0
+    if not now:
+        while timeout < TAKEOVER_DELAY:
+            agent_list_new = list_agents(qclient)
+            agent_dead_list_new = agent_dead_id_list(agent_list_new,
+                                                     'L3 agent')
+            if len(agent_dead_list_new) < len(agent_dead_list):
+                LOG.info("Skipping router failover since an agent came "
+                         "online while ensuring agents offline for %s "
+                         "seconds", TAKEOVER_DELAY)
+                sys.exit(0)
+
+            LOG.info("Agent found offline for seconds=%s but waiting "
+                     "seconds=%s before migration",
+                     timeout, TAKEOVER_DELAY)
+            timeout += 1
+            time.sleep(1)
 
-                try:
-                    if not noop:
-                        migrate_router(qclient, router_id, agent_id, target_id)
-                        migration_count += 1
-                except:
-                    LOG.exception("There was an error migrating a router")
-                    continue
+    migration_count = 0
```

Comment:
This is already initialized at https://github.com/SUSE-Cloud/cookbook-openstack-network/pull/1/files#diff-937ed0b9cba9efd2521c8fd1c9e8416fR268